sync/class.jetpack-sync-module-users.php

```diff
@@ -163,7 +163,18 @@ function save_user_cap_handler( $meta_id, $user_id, $meta_key, $capabilities ) {
 
 	public function enqueue_full_sync_actions( $config ) {
 		global $wpdb;
-		return $this->enqueue_all_ids_as_action( 'jetpack_full_sync_users', $wpdb->usermeta, 'user_id', $this->get_where_sql( $config ) );
+		if ( is_array( $config ) ) {
+			return $this->enqueue_all_ids_as_action( 'jetpack_full_sync_users', $wpdb->usermeta, 'user_id', $this->get_where_sql( $config ) );
+		}
+		/**
+		 * Tells the client to sync all users to the server
+		 *
+		 * @since 4.2.3
+		 *
+		 */
+		do_action( 'jetpack_full_sync_users', 0 );
```

Comment:
Why are we passing `0` here? And if we pass an argument, we should add that to the doc block.